Feat-fix: Make cosmos great again! And few more things.. (moved commits)

snipersir opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
(!!ПЕРЕЗАЛИВ!!)
(Я не мегамозг чтоб смочь в английский в таком объеме, изменений довольно много потому будет пока на русском. Да и с гитом я пока на вы, как могу делаю. И надеюсь не сломал ничего другого, в целом на локалке всё работало. UPD: если заливать то пока на Vega или на Cleo, на мейн вероятно будет рано)

Наполняем и меняем космос вокруг станции, он слишком прост и пуст для игры про космос.
Есть комментарии напротив некоторых изменений и добавлений в коде.

--Файлы--
Добавил иконку 'noop'(полупрозрачная розоватая текстура шашек) в floors.dmi и связал её с 'Turf Passthrough' в файле reader.dm, (теперь в редакторе в картах кроме станции и лаваленда космос отображается как надо, а не в виде напольных плиток, которые перекрывали решетки, каркас, некоторые вещи и в особенности не мешают отображению реальных напольных плиток).

Файлы shuttle.dm, areas.dm изменены. Добавлены зоны шатла, сами шатлы, их платы. Добавлен файл карты spacehotelv1.dm с зонами.

--Карты--
oldstation.dmm
Крупная переработка oldstation aka Theta station. Бета блок стал соответствовать описанию в записке. Карта АПЦ переделана, вшит доступ механика, описание карты изменено. Добавлена карта Мехатроника, доступы: Роботехник. Добавлено ядро слайма в химии, записка с лором есть. Двигатель Синги перенесен на склад в закрытое помещение с доп. запиской, вместо Сингулярности в Прототипной стоит Тесла с нормальной запиской. Кнопочка добавлена для открытия створок прототипов, створки добавлены на окно. Добавлен сейф с ржавым костюмом шахтеров 26 века и запиской внутри. Добавлен переход с Беты на уничтоженную Альфу, на переходе стоит внешняя шахтерская точка. Ниже и левее Беты остатки недостроенной и уничтоженной зоны двигателя под Теслу. Добавлены астеройды. Добавлены мобы Beacon, мили танки в каком-то роде. Текстура ускорителя как на дельте, старая сильно мешала пониманию что куда ставилось. Добавлены латунные инструменты со статами обычных инструментов как спец набор СЕ того времени.
Четвертый член экипажа Медик возвращен на карту, он был в файлах, но не было спавнера, теперь есть. Карте выставлены новые весы, cost = 3, карта уже не маленькая по размерам.

ussp.dmm
На станции советов сделан рабочий шатл, который связан с DJ станцией/постом(как угодно). По лору они имели связь. Между тем, этот шатл уже летал когда-то на другой версии карты, зачем выпилили не ясно. На мостик поставлена консоль шатла.

dj.dmm
На DJ станции прикручен порт для шатла со станции советов. Так же исправлены нерабочие соляры, теперь они не режут глаза и не являются декором. Больше не будет дублироваться в космосе, он по лору лишь один.

spacebar.dmm
Космобар Spacebar частично отредактирован. На нем стоит рабочий шатл до Twin Nexus Hotel, консолей для этого шатла в баре нет. Изменена зона посадки и шлюзов, теперь там можно летать на поде механика. Во входе в бар и астеройд бара, в шлюзах стоят воздушные барьеры, если захочется - теперь можно сделать там базу. Не будет дублироваться в космосе, несколько абсолютно одинаковых бара это бред. Был бы рандомайзер - пожалуйста.

spacehotelv1.dmm
The Twin Nexus Hotel, карта дублирована с других сборок и отредактирована. Больше для рп, чем для экшона. РПшеры вероятно будут рады. Его можно было бы сделать в разы интереснее будь на сборке больше вещей для интерьера. Не дублируется, довольно большой.

vacation.dmm
Астероид с пляжем, дублирован и немного изменен. Приятное и забавное, своеобразное дополнение к космосу. Почему бы и нет?

punishment.dmm
Астероид с похищенным Атмостехником. Он работал так хорошо, что надоел синдикату. Теперь можно найти снарягу Атмостехников в виде одежды, голопроектора, надувных стен и маленького огнетушителя в космосе.

z2.dmm
В секторе ЦК находятся транзитные зоны для шатла. Отредактированы для двух новых шатлов.

## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Больше интерактива в космосе, больше разнообразия и рп, больше возможностей без выкручивания своих нервов. Нормальная работа вещей в редакторе, того же космоса, это надо было сделать давным давно, мы наверно последние кто это делает..
## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif of your feature if you want -->
![2bar](https://user-images.githubusercontent.com/25121334/159734737-7d00be9d-6a1c-4d27-b44e-db316d35107d.png)

![3ussp](https://user-images.githubusercontent.com/25121334/159734750-a371871c-1426-47e9-b554-253f0dc88bd3.png)

![4dj](https://user-images.githubusercontent.com/25121334/159734756-90473e9f-75b4-43e5-ae06-12dba020f182.png)

![5theta](https://user-images.githubusercontent.com/25121334/159734757-1548e860-8445-4680-b6c1-e98376963a36.png)

![6punish](https://user-images.githubusercontent.com/25121334/159734761-e77b864d-7fd1-4fce-aa6d-9bcf5279f2ce.png)

![7vac](https://user-images.githubusercontent.com/25121334/159734766-f2fa95ea-9a74-4f1f-963d-0e3a2ac244b9.png)

![1hotel](https://user-images.githubusercontent.com/25121334/159734767-7c6a701f-f0ce-411b-bafe-52fb1c27eeb7.png)

## Changelog
:cl:
add/добавлено: Added 3 new ruin maps// новые карты добавлены: vacation, punishment, spacehotelv1
tweak/изменено: tweaked 5 maps with CC map "z2" for few shuttles // 5 карт изменено, на ЦК отредактированы места под новые шатлы. The maps // Сами карты: dj outpost, ussp, spacebar, oldstation
fix/исправлено: fixed lost sprite "noop" for map editor // добавил недостоющий спрайт "noop" для облегчения редактирования карт
wip/в_процессе: oldstation, spacehotelv1, spacebar, ussp and dj outpost - will be tracked for feedbacks // ожидаются отзывы по перечисленным картам для доработки и исправлений
/:cl:

snipersir committed:
Feat-fix: Make cosmos great again! Ruin maps changes, some code fixes(?) v2

Moved to another branch to fix things. New maps and some edits for better gameplay. Didn't tweaked yet cuz it wasn't used yet on servers, so no feedback no changes.
Prev desc:
Large map edit. 3 maps added (TN hotel aka spacehotelv1, Vacation, Punishment), 4 edited(Oldstation, Spacebar, DJ Station, USSP + centcom map for 2 new shuttles), 2 new space-ruin working shuttles. The icon 'noop' added to floors.dmi and connected to 'turf passthrough', now mapping other space structures will be better.. 1 new map file spacehotelv1.dm with areas.

snipersir committed:
Feat-fix: Quick DJ station gps tweak, oldstation added mats

Moved to another branch to fix things. Didn't tweaked yet cuz it wasn't used yet on servers, so no feedback no changes.
Prev desc:
DJ station: deleted dev gps that i forgot, oldstation: added materials to mining point

snipersir commented:
Хотел дополнить но редактирование не проходит. Если и заливать всё это то на Клео и/или Вегу. Для мейна рановато, нужны отзывы для правок. И да, это ПЕРЕЗАЛИВ! Вуот.
UPD: судя по онлайну на Веге, где раньше было стабильно вечером от 70, а днем от 50 игроков, возможно придётся заливать на мейн для ускоренного получения фидбэка.

TrashDoxx reviewed (approved):
Требуется дополнительное подтверждение от кого-либо из мапперов. На глаз - всё в норме.

snipersir committed:
Feat-fix: Small cosmetic changes

Added few buttons for doors on spacebar, moved few items and mobs on hotel, theta, punishment maps in different places. Notes edited.

snipersir committed:
Feat-fix: Second small cosmetic changes

Tried to fix spacebar floor under leaving shuttle and removed cost. Prob it doesn't need here. Moved shuttle and port doors at dj station at one atom down, this will be better. Hope this one will be last on the moment, the rest should be good.

dikers93 commented:
ussp.dmm по лору с шатлом связывает только неудачное нападение которое закончилось гибелью всех, а так на глаз нормально

ThaumicNik commented:
Просмотрел. Крупных багов не заметил. Проблема может возникнуть с балансом из-за наполнения локаций довольно полезным лутом, но это уже по решению СА и Азиза. А так всё отлично.
